exampleCourse/questions/element/markdown/question.html

```diff
@@ -1,5 +1,5 @@
 <!--htmlhint tag-pair:false -->
-<!-- htmlhint doesn't understand our <markdown> tag-->
+<!-- htmlhint doesn't understand our markdown tag-->
```

Comment:
Resolves #4231. The `<markdown>` tag was interpreted as the beginning of a markdown block, even inside the comment.